### DISCUSSION TITLE
Close all toasts/snackbars/notifications

### DISCUSSION THREAD

**Discussion by ozgurg** [buefy/buefy#3978] [Ideas]
### Description

Ability to close all shown toasts/snackbars/notifications. (With queue cleaning)

### Why Buefy need this feature

Useful when route changes. Clear page from all shown toasts/snackbars/notifications when route changed.

**Comment by foochris** [comment]
@ozgurg I have a similar request. I have a use case where the `message` length is ~1K and `duration` is based on the text size. Currently, the only method for a user to hide the toast is to refresh the page.

My preference is to use the `Toast` component rather than a workaround such as using a `Modal`.

Ideally, clicking on outside `Toast` or pressing `ESC` would close the component.

cc: @service-paradis

**Comment by jtommy** [comment]
I might save all programmatic instances on a global state and allow to close them by a method, for example: `$buefy.closeAll()`

**Comment by ozgurg** [comment]
> I might save all programmatic instances on a global state and allow to close them by a method, for example: `$buefy.closeAll()`

Yeah, this probably solve my problem.
